src/integrationTest/resources/ccd/callback/create-case/request/valid-exception-without-form-type.json

```diff
@@ -13,6 +13,7 @@
       "journeyClassification": "EXCEPTION",
       "deliveryDate": "2018-01-02T12:34:56.123",
       "openingDate": "2018-01-03T12:34:56.123Z",
+      "envelopeId": "e0c7b48e-624a-4b1f-ba3e-7a0d50d09e80",
```

Comment:
`is_automated_process` and `exception_record_case_type_id` are not included in any of the integration test json files should be add them ?

Reply:
These envelopeIds that I've included are just added to callback data, as they're required there. When it comes to calling transformation endpoint, we have no test verifying what data we've sent to it. In https://tools.hmcts.net/jira/browse/BPS-1336 I've mentioned the need for tests to cover the mapping of callback data to the model representing transformation request. We can also think of integration tests for that (using wiremock).